ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-multicast.yml

```diff
@@ -2,6 +2,13 @@
 
 router_multicast:
   ipv4:
+    rpf:
+      routes:
+        - subnet: 10.10.10.1/32
+          nexthop_ip: 10.9.9.9
+          distance: 1
```

Comment:
For testing the molecule, please keep this value to >1. Wanted to check if its pulling from default filter or the user input.

Reply:
good point.